src/expandable/include.rs

```diff
@@ -55,7 +57,9 @@ pub fn expand_from_filepath(parent_path: &str, benchmark: &mut Benchmark, access
     } else if actions::Assert::is_that_you(item) {
       benchmark.push(Box::new(actions::Assert::new(item, None)));
     } else if actions::Request::is_that_you(item) {
-      benchmark.push(Box::new(actions::Request::new(item, None, None)));
+      for index in 0..times(&item, None) {
```

Comment:
Move this logic into the `request` module

Reply:
Makes sense. Moved it.